src/inja.hpp

```diff
@@ -645,7 +645,7 @@ class Parser {
 	}
 
 	std::map<Parsed::Delimiter, Regex> regex_map_delimiters = {
-		{Parsed::Delimiter::Statement, Regex{"\\{\\%\\s*(.+?)\\s*\\%\\}"}},
+		{Parsed::Delimiter::Statement, Regex{"\\{\\%\\s*(.+?)\\s*(\\%\\}|\\-\\%\\}\\s*\\n)"}},
```

Comment:
Does it handle cases like this:
`
Hello {% if var -%}  {{ var1 }} {% endif %}
`
The space between `-%}` and next `{{` should be removed.
http://jinja.pocoo.org/docs/2.10/templates/#whitespace-control

Reply:
It didn't, but I just pushed an update that does.

Reply:
So what seemed a simple change of making the newline optional in the regex had the unintended side effect of also removing whitespace at the beginning of the next line, after the newline.  I clearly have an imperfect understanding of ECMAScript regexes.
I have reverted back to the original commit that just removes whitespace up to and including the newline.  The intent of this PR is to address Issue #49, and the original commit does that.

Reply:
Jinja also removes the whitespace in the next line. So I would just delete the `\\n` in the regex, and then we are good to go.